src/api/schema.ts

```diff
@@ -528,6 +542,16 @@ export const lists = {
     },
     access: allowAll,
   }),
+  BusinessContactDetail: list({
```

Comment:
What about `BusinessContact` instead? What's the "Detail" for?

Reply:
discussed.  Initially had agreed on `BusinessContactDetail`